addons/l10n_be_edi/__manifest__.py

```diff
@@ -11,7 +11,7 @@
 (cf the `official announce <http://www.prweb.com/releases/2016/01/prweb13186919.htm>`_).
 Belgian e-invoicing uses the UBL 2.0 using the e-fff protocol.
     """,
-    'depends': ['account', 'l10n_be', 'account_facturx'],
+    'depends': ['account', 'l10n_be', 'account_invoice_import'],
```

Comment:
you can remove the dependency of `account` since this is an existing dependency of `account_invoice_import`